### PR TITLE
Rewrite lottie component to custom

### PR DESCRIPTION
**Pull request by Programistich**
**Background**

On bottom bar freeze animation

**Changes**

* Replace Lottie on two images

**Test plan**

Open application and try connect

### CODE REVIEW

File: components/bottombar/api/src/test/kotlin/com/flipperdevices/bottombar/TabStateTest.kt [hunk 1]
```diff
@@ -12,15 +12,19 @@ class TabStateTest {
         notSelectedIcon = 2,
         text = "TabState Animated 1",
         selectedColor = Color(0xFF111111),
-        unselectedColor = Color(0xFF222222)
+        unselectedColor = Color(0xFF222222),
+        selectedBackground = 0,
```
Comment (LionZXY) [inline components/bottombar/api/src/test/kotlin/com/flipperdevices/bottombar/TabStateTest.kt:16]: What is it?

File: components/bottombar/api/src/main/java/com/flipperdevices/bottombar/model/TabState.kt [hunk 2]
```diff
@@ -53,9 +52,12 @@ sealed class TabState private constructor(
         }
     }
 
+    @Suppress("LongParameterList")
     class Animated(
-        @RawRes val selectedIcon: Int,
-        @RawRes val notSelectedIcon: Int,
+        @DrawableRes val selectedIcon: Int,
+        @DrawableRes val selectedBackground: Int,
+        @DrawableRes val notSelectedIcon: Int,
```
Comment (LionZXY) [inline components/bottombar/api/src/main/java/com/flipperdevices/bottombar/model/TabState.kt:59]: Maybe mark it as `null`?

Comment (Programistich) [reply components/bottombar/api/src/main/java/com/flipperdevices/bottombar/model/TabState.kt:59]: In what case its help?

File: components/bottombar/api/src/test/kotlin/com/flipperdevices/bottombar/TabStateTest.kt (hunk 1 above)
Comment (Programistich) [reply components/bottombar/api/src/test/kotlin/com/flipperdevices/bottombar/TabStateTest.kt:16]: It's test, I think necessary in test

**Pull request merged**